conans/client/conan_api.py

```diff
@@ -200,9 +201,11 @@ def __init__(self, cache_folder, user_io, http_requester=None, runner=None, quie
 
         self.proxy = ConanProxy(self.cache, self.out, self.remote_manager)
         self.range_resolver = RangeResolver(self.cache, self.remote_manager)
-        self.python_requires = ConanPythonRequire(self.proxy, self.range_resolver)
+        self.generators = GeneratorManager()
```

Comment:
Can we call this one `generator_manager`?

 ```suggestion
         self.generator_manager = GeneratorManager()
```